### ISSUE TITLE
default capture device doesn't exist on macOS

### ISSUE THREAD

**Issue by picatz** [sdaubert/packetgen#76]
👋 Hello @sdaubert, I really hope you're doing well!

This morning I decided to run `rspec` on this repository and came across an error on macOS:
```
../packetgen/lib/packetgen/capture.rb:65:in `open_live': lo: No such device exists (BIOCSETIF failed: Device not configured) (RuntimeError)
```

I believe this to be attributed to [the default capture interface](https://github.com/sdaubert/packetgen/blob/master/lib/packetgen/capture.rb#L48) `lo`. On macOS, perhaps at the very least on my laptop, the loopback interface is named `lo0`.

## Interfacez
I wrote a gem a little while back called [`interfacez`](https://github.com/picatz/interfacez). This gem uses the Ruby standard library under the hood and provides a clean semantics for this behavior I think we're looking for. Moreover, I really think it could be a replacement for [`NetworkInterface`](https://github.com/rapid7/network_interface) ( written by Rapid7, with lots of slow C bindings ) and event `Pcap.lookupdev`. 

#### Example Usage
```ruby
require 'interfacez'

# check if loopback exists
Interfacez.loopback?
# => true

# get loopback
Interfacez.loopback
# => "lo0"

# get first network interface to capture on ( lookupdev )
Interfacez.default
# => "en0"
```

**Comment by sdaubert** [comment]
hey! that's a great gem :tada:

I will have a look at it, when i will have some time :confused:

**Comment by picatz** [comment]
Thanks! 🙇 

If you are interested, I can spend some time this week replacing `network_interface` with `interfacez`.

**Comment by sdaubert** [comment]
Yes, this could help.

**Comment by sdaubert** [comment]
@picatz may you add a method to get hwaddr (ie MAC address) on a given interface? This should help to replace NetworkInterface by Interfacez.

**Comment by picatz** [comment]
@sdaubert I think I should be able to implement that. 👍

**Comment by picatz** [comment]
@sdaubert Got those issues addressed. Going to release new gem version. 🎉

**Comment by sdaubert** [comment]
Should be fixed by #85

**Comment by picatz** [comment]
@sdaubert That's awesome! 🎉 ❤️ ✨
